docs/guides/working_with_data/dataframes.md

```diff
@@ -134,12 +174,29 @@ transformed_df.value
 
 ///
 
-<div align="center">
-<figure>
-<img src="/_static/docs-dataframe-transform-code.png"/>
-<figcaption>Copy the code of the transformation</figcaption>
-</figure>
-</div>
```

Comment:
personally i like the `gif` (they are quick to load and demo what to do).

could we put the demos in another tab? `🐻‍❄️ polars | 🐼 pandas | ⚡ live example`

Reply:
I'll put the image back, it's not a gif, but I don't really recall why it got removed. I guess I might've imagined that we no longer needed it now that there was a live demo?